src/services/health-check/presenter-result-builder.php

```diff
@@ -0,0 +1,207 @@
+<?php
+
+namespace Yoast\WP\SEO\Services\Health_Check;
+
+/**
+ * Provides an interface to build WordPress-friendly health check results.
+ */
+class Presenter_Result_Builder {
+
+	/**
+	 * Passed health check.
+	 */
+	const STATUS_GOOD = 'good';
+
+	/**
+	 * Changes are recommended but not necessary.
+	 */
+	const STATUS_RECOMMENDED = 'recommended';
+
+	/**
+	 * Significant issues that the user should consider fixing.
+	 */
+	const STATUS_CRITICAL = 'critical';
+
+	/**
+	 * The user-facing label.
+	 *
+	 * @var string
+	 */
+	private $label = '';
+
+	/**
+	 * The identifier that WordPress uses for the health check.
+	 *
+	 * @var string
+	 */
+	private $test_identifier = '';
+
+	/**
+	 * The test status (good, recommended, critical).
+	 *
+	 * @var string
+	 */
+	private $status = '';
+
+	/**
+	 * The short description for the result.
+	 *
+	 * @var string
+	 */
+	private $description = '';
+
+	/**
+	 * Actions that the user can take to solve the health check result.
+	 *
+	 * @var string
+	 */
+	private $actions = '';
+
+	/**
+	 * Sets the label for the health check that the user can see.
+	 *
+	 * @param  string $label The label that the user can see.
+	 * @return Presenter_Result_Builder This builder
+	 */
+	public function set_label( $label ) {
+		$this->label = $label;
+		return $this;
+	}
+
+	/**
+	 * Sets the name for the test that the plugin uses to identify the test.
+	 *
+	 * @param  string $test_identifier The identifier for the health check.
+	 * @return Presenter_Result_Builder This builder.
+	 */
+	public function set_test_identifier( $test_identifier ) {
+		$this->test_identifier = $test_identifier;
+		return $this;
+	}
+
+	/**
+	 * Sets the status of the test result to GOOD (green label).
+	 *
+	 * @return Presenter_Result_Builder This builder.
+	 */
+	public function set_status_good() {
+		$this->status = self::STATUS_GOOD;
```

Comment:
```suggestion
		$this->status = self::STATUS_GOOD; // these consts needs to be translatable.
```
see [example](https://github.com/Yoast/wordpress-seo/blob/f3a2dde76cb31a8feb5adea944f2044e5bfd7006/admin/views/licenses.php#L22)

Reply:
not translatable, it should map to WordPress internal constants.